src/main/resources/component-biz.conf

```diff
@@ -654,9 +655,10 @@ security {
             permission-system-load = true
             permission-tasks = true
             permission-manage-all-processes = true
-            feature-provide-jobs = true
```

Comment:
Not Subject of this commit ?

Reply:
yep, this was a left-over of a pervious refactoring - the feature is long gone...